Character.cpp

```diff
@@ -5,8 +5,8 @@
 #include "DefensiveItem.h"
 #include "HelpfulItem.h"
 
-Character::Character(int hp, int armor_, int attackDamage_ ) :
-    hitPoints(hp),
+Character::Character(int hp_, int armor_, int attackDamage_ ) :
+    hitPoints(hp_),
```

Comment:
This change was not needed
- [ ] revert

Reply:
Did you mean: Change `hp_` to `hp`?